tests/e2e/specs/setup-mc/step-4-complete-campaign.test.js

```diff
@@ -140,6 +140,27 @@ test.describe( 'Complete your campaign', () => {
 		).toBeVisible();
 	} );
 
+	test.describe( 'Free Ad Credit', () => {
+		test( 'shoud not see the Free Ad Credit section if the account is not eligible', async () => {
+			await expect(
+				page.getByText(
+					'Spend $500 to get $500 in Google Ads credits!'
+				)
+			).not.toBeVisible();
+		} );
+
+		test( 'should see the Free Ad Credit section if the account is eligible', async () => {
+			await setupAdsAccountPage.mockAdsSubAccountConnected();
+			await completeCampaign.goto();
+
+			await expect(
+				page.getByText(
+					'Spend $500 to get $500 in Google Ads credits!'
+				)
+			).toBeVisible();
+		} );
+	} );
+
```

Comment:
@asvinb While this seems to be passing, it looks like those are not working as expected. For the first test `shoud not see the Free Ad Credit section if the account is not eligible`, page is not loading at all and there is always a spinner on the page, thus it is passing the assertion that text is not existing.

This can be better validated with the `--ui` option in e2e tests.

```
npm run test:e2e -- step-4-complete-campaign.test.js --grep "Free Ad Credit" --ui
```

Reply:
@ankitrox I made a small tweak to the PR and added 
`await completeCampaign.goto();`
I checked again and there's indeed a spinner but as you hover on the "timeline", you can see the loaded page:
<img width="1296" alt="image" src="https://github.com/user-attachments/assets/4f794d15-87ec-4813-81fa-08e13ec02238">

Additionally, to make sure, I added:
```js
await expect(
	page.getByRole( 'heading', {
		name: 'Create a campaign to advertise your products',
	} )
).toBeVisible();
```

just before the "Spend $500 to get $500 in Google Ads credits!" test and the test passes. Can you kindly check and let me know please?